Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
```

Comment:
I also just wanted to inform you that using `make install` would be significant slower than our current approve (Especially because you cared a lot about performance in the postinstall part). However it is correct that the additional time required for `npm pack` is insignificant compared to the time it takes to compile node from source.

Also the more important part is, that the npm@5.2.0 upgrade would require changes because of `npx`.

> As upstream has said, it was effectively already doing that before.

This is only correct if before is referring to the pre npm@5 state of this formula and not to the current state, because we atm are installing the already packed npm registry tarball we've previously downloaded.